server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPServer.java

```diff
@@ -90,10 +95,19 @@ public void start() {
                     .orElse(RANDOM_PORT))
                 .handle((request, response) -> handleVersionRoute(request).handleRequest(request, response))
                 .wiretap(wireTapEnabled())
+                .metrics(true, Function.identity())
```

Comment:
```suggestion
                .metrics(ENABLED, Function.identity())
```

And out of question what does `Function.identity()` allow to do?

Reply:
Could i be configurable? Eg if perf impact is high...